stable/polaris/templates/dashboard.rbac.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.dashboard.enable .Values.rbac.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "polaris.fullname" . }}-dashboard
+  namespace: {{ .Release.Namespace }}
```

Comment:
Would it be possible to put these namespace blocks inside of a `templateOnly` if statement?  Putting them in when deploying with Helm is not a great practice

Reply:
Sure thing, done

Reply:
Thanks!

Reply:
Thanks, but I think I wasn't clear.  I think all the namespace specifications should be done like that (there's several more throughout the chart)